test/test_meta.py

```diff
@@ -187,15 +188,79 @@ def test_tensor_outlives_converter(self):
         del m
         self.assertIs(ref(), None)
 
+aten = torch.ops.aten
+
 CHECK_STRIDES = {
     torch.Tensor.__getitem__,
-    torch.ops.aten.index_put,
-    torch.ops.aten.index_add,
+}
+
+CHECK_STRIDES_SKIPS = {
+    aten._conj_physical.default,
+    aten._fft_c2c.default,
+    aten._fft_c2r.default,
+    aten._fft_r2c.default,
```

Comment:
nit: we could make this expected failures by running it once with check_strides = True, asserting it fails if it's in this list, then running it again with check_strides=False

Reply:
XFAIL in test harness is asserting for every sample... 
However, for the stride tests, not all sample will fail, so we still need to SKIP... :(